package.json

```diff
@@ -13,7 +13,9 @@
     "lint:all": "nx run-many --all --target=lint",
     "e2e:all": "nx run-many --all --target=e2e",
     "vegacapsule": "vegacapsule network bootstrap --config-path=../frontend-monorepo/vegacapsule/config.hcl",
-    "release": "git checkout develop ; git pull ; node scripts/make-release.js"
+    "release": "git checkout develop ; git pull ; node scripts/make-release.js",
+    "trading:test": "cd apps/trading/e2e && poetry run pytest -k",
```

Comment:
Nice! 
Do we run this after:  

```bash
yarn nx build trading ./docker/prepare-dist.sh docker build -f docker/node-outside-docker.Dockerfile --build-arg APP=trading --build-arg ENV_NAME=stagnet1 -t vegaprotocol/trading:latest .
```

Reply:
Yeah you will need to run that after.
You have two options:
`yarn trading:test -- "test_name" -s `
run this from anywhere
or
`poetry run pytest -k "test_name" -s`
run this from within apps/trading/e2e